Add confluent/jitpack repositories for trino-kafka

PChou opened the pull request:
Without repositories configuration, maven can not fetching some dependencies

Praveen2112 commented:
Can you please let us know what are the dependencies.

Praveen2112 commented on plugin/trino-kafka/pom.xml:
```diff
@@ -17,6 +17,17 @@
+    <repositories>
+        <repository>
+            <id>confluent</id>
+            <url>https://packages.confluent.io/maven/</url>
```
This is already added in root pom.xml with snapshot disabled

PChou replied:
ok, got it

PChou commented:
@Praveen2112 io.confluent:kafka-json-schema-serializer which finally bring com.github.everit-org.json-schema:org.everit.json.schema:1.12.1

https://github.com/everit-org/json-schema#maven-installation

Praveen2112 commented:
From where do we refer these dependencies (kafka-json-schema-serializer) ? Is it required in the runtime or ? I'm not sure if we could a way to configure the serializer/deserializers in Kafka connector (currently it uses `ByteArraySerializer`) and implement a decoder based on the data format.

findepi commented:
@PChou before you spend more time on this, can you describe the problem you're solving?

Also, the PR seems to now include changes in Elasticsearch connector which are likely unrelated. Can you please move them to a separate PR?
I recommend _not_ making PR from the `master` branch, it seems convenient at first, but can quickly become unwieldy.

PChou commented:
@findepi Sorry, the change of elasticsearch connector is something else, I forgot to separate it. The problem I encountered was that when I tried to build trino, I found that the `trino-kafka` module could not be successfully built because there were dependencies that could not be found. After many attempts, it still failed. Finally, I found `com.github.everit-org. json-schema:org.everit.json.schema:1.12.1` could not be found in the public maven repository, and I solved it by adding `https://jitpack.io`. I think other new developer may also encounter the same problem.

hashhar commented:
This shouldn't be needed since `io.confluent:kafka-json-schema-serializer` already defines the `jitpack` repo.

Do you see `jitpack.io` in the output of `mvn org.apache.maven.plugins:maven-dependency-plugin:2.9:list-repositories -pl plugin/trino-kafka`?

PChou commented:
@hashhar As you said, `io.confluent:kafka-json-schema-serializer` does already define the `jitpack` repo. Thanks for the guidance, I will close this PR.

The reason why the build failed at the beginning was that i set up the mirror repo for my maven and did not configure jitpack.io to the exclusion list. but after that, I added jitpack.io in pom.xml and also added jitpack.io to the exclusion list.

I configure the mirror repo and exclusion like this:
```
<mirrors>
         <mirror>
             <id>aliyunmaven</id>
             <mirrorOf>*,!confluent,!redshift,!jitpack.io</mirrorOf>
             <name>aliyunmaven</name>
             <url>https://maven.aliyun.com/repository/public</url>
         </mirror>
     </mirrors>
 ```
It would be better if you can write some documents to guide new developers
